dask/dataframe/core.py

```diff
@@ -2193,18 +2213,64 @@ def std(
             )
             return handle_out(out, result)
         else:
```

Comment:
Another de-nesting idea: this doesn't really need an else block since the `axis==1` case has already returned.